pkg/webhook/validator/pipeline.go

```diff
@@ -82,6 +82,10 @@ func (v *pipelineValidator) checkISBSVCExists(ctx context.Context, isbSvcName st
 	if err != nil {
 		return err
 	}
+	// Check if they have the same instance annotation
```

Comment:
can we rename the method to `checkISBSVCIsValid`? and also unit test it.

Reply:
done.